wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitUrl.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.git.impl.nativegit;
+package org.eclipse.che.api.git;
```

Comment:
I don't know. Maybe it will be OK to find some commons title template. BTW It doesn't have any relation to GitUrl.